src/main/topLevelPages/myDashboard/newVirtualContributorWizard/useNewVirtualContributorProps.ts

```
@@ -0,0 +1,56 @@
+        roleSet: {
+          id: string;
+          authorization?: {
+            myPrivileges?: AuthorizationPrivilege[] | undefined;
+          };
+        };
+      };
+    }>;
+  }>;
+}
+
+interface NewVirtualContributorWizardProps {}
```

Biome

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)